### ISSUE TITLE
Add support for `signBytes` calls

### ISSUE THREAD

**Issue by laurogripa** [ChainSafe/cypress-polkadot-wallet#49]
I need to test a call made with papi's `signBytes`, it would be nice to have support for it, maybe something like:
- `cy.getSignRequests()` and `cy.approveSign()`
-  or: just change `cy.approveAuth()` to support signatures, since `cy.getAuthRequests()` already tracks the call `id`

**Comment by laurogripa** [comment]
Ok, so after tinkering a bit, I realized the plugin actually already supports papi's `signBytes` because under the hood it's basically a wrapper around pjs' `signRaw` (which is already supported)

What was actually wrong was that I was using an older version of papi where `connectInjectedExtension` had only 1 argument (`extensionName`) and didn't allow for passing the `dappName`, this meant `origin` was undefined and generating a weird side effect where the signature request was getting added to `authRequests` but without a way of approving it.

After updating papi to a version above 1.6.0, where @Tbaut introduced the possibility to pass `dappName` as an argument to `connectInjectedExtension` (https://github.com/polkadot-api/polkadot-api/pull/791), the issue was fixed. After fixing that, the signature request was added to the `txRequests` queue instead and could be approved with `approveTx(id)`.
See this PR for reference: https://github.com/PolkadotEducation/app/pull/173/

I'm closing the issue since the plugin actually supports such calls, but just opened a PR (#50) adding a note to the README to encourage anyone testing with papi to use 1.6.0 or above and set up `dappName`
